src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -3843,6 +3843,8 @@ pub const TCP_INFO: ::c_int = 32;
 pub const TCP_CONGESTION: ::c_int = 64;
 pub const TCP_CCALGOOPT: ::c_int = 65;
 pub const TCP_MAXUNACKTIME: ::c_int = 68;
+#[doc(hidden)]
+#[deprecated(since = "0.2.160", note = "Removed in FreeBSD 15")]
 pub const TCP_MAXPEAKRATE: ::c_int = 69;
```

Comment:
I think it would be better to remove `#[doc(hidden)]` for now so it doesn't disappear entirely. We can add it back in a couple versions.

Reply:
Ok.  Though note that the sockopt was never actually implemented within the kernel, so no program that tried to use it could've possibly worked.